.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -10,3 +10,6 @@
  - [ ] manually test the debug page
  - [ ] tagged `@mapbox/studio` and/or `@mapbox/maps-design` if this PR includes style spec or visual changes
  - [ ] tagged `@mapbox/gl-native` if this PR includes shader changes or needs a native port
+ - [ ] include changelog info
+    - [ ] add a changelog label ('bug', 'feature', 'docs', etc). Use the label 'skip changelog' to hide this PR in the changelog
+    - [ ] include a changelog entry. The PR title is used or optionaly a tagged `<changelog>...</changelog>` section in the body.
```

Comment:
Please include a `<changelog>...</changelog>` section here or outside of the checklist so that it's easier to fill it out while authoring the PR.

Reply:
Sorry, I have the tags here with the ellipses instead of content. You want a different formatting here?

About the optional tagged body section, people seemed confident the vast majority of the time we'd just use a short entry text in the title. I understood this feature as an escape hatch for when we need longer entries (which should be infrequent). If this is right I'm inclined to leave it in the checklist, but always happy to be convinced.

Reply:
If we leave it in the checklist (my preference) am I right the formatting you'd like to see leaves the `- [] <changelog>...</changelog>` on its own line? Do I misunderstand?

Reply:
> - [] `<changelog>...</changelog>` on its own line?

That works too